integration_tests/e2e/induction/updateInPrisonInterests.cy.ts

```diff
@@ -4,6 +4,9 @@ import InPrisonWorkValue from '../../../server/enums/inPrisonWorkValue'
 import EducationAndTrainingPage from '../../pages/overview/EducationAndTrainingPage'
```

Comment:
I've renamed this file to `updateInPrisonInterests` to cover in-prison work and training (to follow). Otherwise, it's going to become very large.

Reply:
Hmmmm - might need a quick chat on that ....

Reply:
Actually, I've moved some of the specific tests out into a separate file.